frontend/src/App.vue

```diff
@@ -1,43 +1,17 @@
 <template>
-  <div>
-    <h4>Hello Friendos</h4>
-    Latitude: {{ currPosition.lat.toFixed(2) }},
-    Longitude: {{ currPosition.lng.toFixed(2) }}
-  </div>
-  <div ref="mapDiv" style="width:100%; height:80vh" />
+  <hello-world></hello-world>
```

Comment:
checking, do we need the hello-world component here?

Reply:
nope, removed!